src/material/core/option/option.ts

```diff
@@ -196,13 +196,20 @@ export class _MatOptionBase<T = any> implements FocusableOption, AfterViewChecke
   }
 
   /**
-   * Gets the `aria-selected` value for the option. We explicitly omit the `aria-selected`
-   * attribute from single-selection, unselected options. Including the `aria-selected="false"`
-   * attributes adds a significant amount of noise to screen-reader users without providing useful
-   * information.
+   * The ARIA selected applied to the option. Conforms to WAI ARIA best practices for listbox
+   * interaction patterns.
+   *
+   * From [WAI ARIA Listbox authoring practices guide](
+   * https://www.w3.org/WAI/ARIA/apg/patterns/listbox/):
+   *  "If any options are selected, each selected option has either aria-selected or aria-checked
+   *  set to true. All options that are selectable but not selected have either aria-selected or
+   *  aria-checked set to false."
+   *
+   * Set `aria-selected="false"` on not-selected listbox options that are selectable to fix
+   * VoiceOver reading every option as "selected" (#25736).
    */
   _getAriaSelected(): boolean | null {
```

Comment:
Should we just get rid of this method?

Reply:
What I'm trying to accomplish is have it clearly documented *why* we're doing aria-selected this way.  That way future readers of the code can have the context they need.

I don't feel strongly about having this be it's own function. I can move the code to the host binding and put the comment at the top of the host binding if that's preferred.

Reply:
done. I moved the code and the comment to the host binding.